### PR TITLE
Added Week 1 progress

### PR DESCRIPTION
**Pull request by premsaivarmachekuri**
## Project

BeckN in a box

### Changes

- [x] Added my Week 1 Progress
- [x] Added images of my 1 milestone

### CODE REVIEW

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]
Please follow the format

**Comment by pvsyvarmachekuri** [comment]
> Please follow the format

Can you say the changes to be made

**Comment by Vivek-Kumar-Shrivastav** [comment]
you have to make two changes:
1. Give your image files a meaningful name, not just "image.png"
2. In each week change the author name by your name.

File: docs/2023/BeckN/Beckn in a box/updates/image-1.png
Comment (Vivek-Kumar-Shrivastav) [inline docs/2023/BeckN/Beckn in a box/updates/image-1.png:1]: change the image file name

File: docs/2023/BeckN/Beckn in a box/updates/image.png
Comment (Vivek-Kumar-Shrivastav) [inline docs/2023/BeckN/Beckn in a box/updates/image.png:1]: change the image file name

**Pull request closed without merging**